OpenSim/Actuators/osimActuatorsDLL.cpp

```diff
@@ -82,7 +82,7 @@ BOOL APIENTRY DllMain( HANDLE hModule,
 
     return TRUE;
 }
-#elif defined(__linux__)
+#else
```

Comment:
Without these PR, `Plugin_Attach()` and `Plugin_Detach()` are invoked only on Windows and Linux. Now, they are also invoked on macOS.